ChangeLog

```diff
@@ -1,5 +1,14 @@
 mbed TLS ChangeLog (Sorted per branch, date)
 
+= mbed TLS x.x.x branch released xxxx-xx-xx
+
+Bugfix
+   * Ignore iv in mbedtls_cipher_set_iv() when the cipher mode is MBEDTLS_MODE_ECB
+     Fix for #1091 raised by ezdevelop
+
+Changes
+   * Add tests for the mbedtls_cipher_crypt() API
```

Comment:
I am not sure if its necessary to add a changelog entry for this.

Reply:
I wasn't sure, but since this is a Bug fix as well, I added one